sycl/include/sycl/access/access.hpp

```diff
@@ -20,15 +20,15 @@ inline namespace _V1 {
 namespace access {
 
 enum class target {
-  global_buffer __SYCL2020_DEPRECATED("use 'target::device' instead") = 2014,
+  device = 2014,
```

Comment:
Made this change because of a peculiarity of the `PrettyPrinter` in the frontend. Even if the user says something like target::device in source code, the pretty printer seems to print that as `target::global_buffer` which results in warnings due to deprecation every time compilation occurs. It seems to default to the first enum that defined that value(in this case the value 2014). AFAIK, none of the flags currently present in `PrintingPolicy` addressed this problem so I made this change instead which I personally don't find invasive at all. It does make sense to have the deprecated enum be defined in terms of the in-effect enum.

Reply:
Given that we ended up modifying PrintingPolicy , should that change be reverted?

Reply:
The PrintingPolicy change fixed another issue and not this. This still remains where the Printer will print the very first declaration of the number 2014 even if we explicitly refer to it by the `device` enum value.